pulsar-client-cpp/lib/MultiTopicsConsumerImpl.cc

```diff
@@ -749,3 +749,15 @@ bool MultiTopicsConsumerImpl::isConnected() const {
     }
     return true;
 }
+
+uint64_t MultiTopicsConsumerImpl::getNumberOfConnectedConsumer() {
+    Lock lock(mutex_);
+    uint64_t numberOfConnectedConsumer = 0;
+    const auto consumers = consumers_;
```

Comment:
Call `lock.unlock()` before this line because after this line only the local variable `consumers` is accessed and the lock is not required anymore.